src/playbacks/html5_video/html5_video.js

```diff
@@ -95,6 +95,18 @@ export default class HTML5Video extends Playback {
     return this._isBuffering
   }
 
+  get isLive() {
+    return this.getPlaybackType() === Playback.LIVE
+  }
+
+  get dvrEnabled() {
+    return this.getDuration() >= this._minDvrSize && this.isLive
+  }
+
+  get hasMinimumDVRConfig() {
+    return this.options.playback && this.options.playback.minimumDvrSize && typeof (this.options.playback.minimumDvrSize) !== 'undefined'
```

Comment:
What do you think about strict the accepted type here?

Reply:
Sure. Done in a4b1aa6